lib/phlex/compiler/class_compiler.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Phlex::Compiler::ClassCompiler < Prism::Visitor
+	def initialize(compiler)
+		@compiler = compiler
+	end
+
+	def compile(node)
+		visit_all(node.child_nodes)
+	end
+
+	def visit_def_node(node)
+		Phlex::Compiler::MethodCompiler.new(@compiler).compile(node)
+	end
```

Comment:
We may need to check the receiver of this  method definition. Not sure if this node would be triggered for something like `def Foo.bar`